pkg/application/service/base/base_service.go

```diff
@@ -0,0 +1,11 @@
+package base
+
+import "github.com/codeready-toolchain/registration-service/pkg/application/service/context"
+
+type BaseService struct {
+	context.ServiceContext
+}
+
+func NewBaseService(context context.ServiceContext) BaseService {
+	return BaseService{context}
+}
```

Comment:
It does just hold the service context for now... give me some time to think over whether we really need this.

Reply:
After some thought I'd like to keep this here as a formality.  Every single service requires the `ServiceContext`, and it's good practice to extend the `BaseService` just in case there's some framework-level changes required in the future.

Reply:
Ok